staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/controller.go

```diff
@@ -114,6 +114,7 @@ func (c *Controller) Run(staticSpec *spec.Swagger, openAPIService *handler.OpenA
 		utilruntime.HandleError(fmt.Errorf("failed to initially create OpenAPI spec for CRDs: %v", err))
 		return
 	}
+	close(openapiControllerSyncedCh)
```

Comment:
does this mean aggregated APIServices are also synched? Or only CRDs?

Reply:
That is a good question. But I think, this is only for CRDs.

Reply:
Can you check and add a comment?